torch/_inductor/decomposition.py

```diff
@@ -411,6 +411,9 @@ def nan_to_num(x, nan=0.0, posinf=None, neginf=None):
 def all(input):
     return torch.logical_not(torch.any(torch.logical_not(input)))
 
+@register_decomposition([aten.expm1.default])
+def expm1(input):
```

Comment:
We need to do a lowering with a libdevice/std function (similar to erf), this is not enough accuracy. While you are at it, can you also delete log1p decomp from inductor and do the same lowering thing? With the tests for values close to one to check accuracy.

Reply:
I looked, and `expm1` isn't listed in the [ptx instruction set](https://docs.nvidia.com/cuda/parallel-thread-execution/index.html). What am I missing ?

Reply:
nvm it's here https://docs.nvidia.com/cuda/libdevice-users-guide/__nv_expm1.html

Reply:
It's not in ptx, it's in triton's wrapper for libdevice that we call like this https://github.com/pytorch/pytorch/blob/master/torch/_inductor/codegen/triton.py#L211 for a lot of functions that are not in ptx set.